ga4gh/client.py

```diff
@@ -371,30 +389,71 @@ def searchReferences(
         return self._runSearchRequest(
             request, "references", protocol.SearchReferencesResponse)
 
-    def searchCallSets(self, variantSetId, name=None):
+    def searchCallSets(self, variantSetId, name=None, bioSampleId=None):
         """
         Returns an iterator over the CallSets fulfilling the specified
         conditions from the specified VariantSet.
 
         :param str name: Only CallSets matching the specified name will
             be returned.
+        :param str bioSampleId: Returns only callsets with the given
+            Id.
         :return: An iterator over the :class:`ga4gh.protocol.CallSet`
             objects defined by the query parameters.
         """
         request = protocol.SearchCallSetsRequest()
         request.variantSetId = variantSetId
         request.name = name
+        request.bioSampleId = bioSampleId
         request.pageSize = self._pageSize
         return self._runSearchRequest(
             request, "callsets", protocol.SearchCallSetsResponse)
 
-    def searchReadGroupSets(self, datasetId, name=None):
+    def searchBioSamples(self, datasetId, name=None, individualId=None):
+        """
+        Returns an iterator over the BioSamples optionally matching
+        a name or individualId for the given datasetId.
+
+        :param str name: Only BioSamples given this name are returned
```

Comment:
``Only BioSamples given this name`` reads a bit weirdly. "Only BioSamples with this name" perhaps?